data/yaml/page-content/chat.yaml

```diff
@@ -17,7 +17,7 @@ sections:
     releaseStage: ''
     callToAction:
       text: View setup instructions
-      href: '/docs/chat/setup'
+      href: '/chat/setup'
```

Comment:
This is all gonna change in a min with all links formatted with `/docs/` in place (the ticket in my in progress queue) but if this fixes it in this moment then sure

Reply:
yeah, this is how it should have been done in the first place!

i'm curious, how will it work going forward, just add `/docs` to all the links? and locally?